_docs/app-metrics.md

```diff
@@ -1,5 +1,57 @@
-# App Metrics
 
+# Status usage data
+Starting release 1.14, the Status mobile app asks to share privacy-preserving, end-to-end encrypted data about how you use Status. Data is only ever shared if you opt in to doing so, you can review all data before it is sent and it is shared over Status' peer-to-peer network (Waku), just like a 1:1 message.
+
+Sharing data is strictly opt-in, and can be changed at any times in Settings in the app.
+
+### What is shared (Opt-in)
+Status is an open-source platform. To verify that the app only collects and shares the events and metadata listed below, you can view the rules set to store metrics data in the [source code](https://github.com/status-im/status-go/blob/develop/appmetrics/validators.go).
+
+- Your interactions with the app like clicks and screenviews
+- Background activity and internal processes
+- Settings and preferences
+
+In detail this means the app can collect and share:
+- Navigation events
+- Screens that are navigated to
+- Action events (for example button clicks, list swipes, switch toggles, etc)
+- The time events are created
+- Operating System
+- App version
+- Batch ID
```

Comment:
what's a batch id?

Reply:
Data is stored and send in batches to limit data that is stored over time. A batch is a collection of data that is removed after transmission. Batch ID is there to view events against an interval. High number of events in a batch implies active use, low number of events in a batch implies passive use. It's somewhat similar to a Session ID but not the same as we don't have a definition of a session that triggers transmission.

cc @Samyoul @shivekkhurana to clarify if needed

Reply:
The batch id is updated each time a user logs in. So it gives us an idea about what the user did in a particular sitting without exposing any identity.